expr/boolean.go

```diff
@@ -390,11 +391,46 @@ func Comparison(op string, literal ast.Literal) (Boolean, error) {
 		return CompareBool(op, v)
 	case float64: //XXX
 		return CompareFloat64(op, v)
-	case zng.Bstring: //XXX
+	case Bstring: //XXX
 		return CompareBstring(op, v)
 	case int64:
 		return CompareInt64(op, v)
 	default:
 		return nil, fmt.Errorf("unknown type of constant: %s (%T)", literal.Type, v)
 	}
 }
+
+//XXX this shoulw all go away ?
+
+type Port uint32
+type Bstring []byte
```

Comment:
No need to export this, but if it's really going away, just leave it.
```suggestion
type bstring []byte
```